src/webapp_vite/src/ArmView/ArmAngleControl.tsx

```diff
@@ -140,7 +136,9 @@ export function ArmControl({
     return (
         <div className={st.container}>
             <div className={st.currentAngle}>
-                {parseFloat(currentAngle.toString()).toFixed(3)}&deg;
+                {!!currentAngle &&
```

Comment:
This shouldn't technically be necessary, but something about the way vite loads components is causing the previous list of controls to still be loaded after arm_config_selected changed